backends/platform/atari/readme.txt

```diff
@@ -0,0 +1,422 @@
+
+By the way, the vsync flag in Global Options affects also the overlay rendering
+(with all the pitfalls which apply to the single buffering mode)
+
+Slow GUI
+~~~~~~~~
+
+Themes handling is quite slow - each theme must be depacked, each one contains
+quite a few XML files to parse and quite a few images to load/convert. Best is
+to use the built-in one. A compromise solution is to depack the theme in an
+equally named directory. It will be loaded, too but without the depacking
+overhead.
+
+
+Known issues
+------------
+
+- aspect ratio correction works on RGB only (yet)
+
+- SuperVidel's DVI output is stretched when in 320x200 or 640x400; I'll  wait
+  for other people's experiences, maybe only my LCD is so lame.
+
+- adding a game in TOS and loading it in FreeMiNT (and vice versa) generates
+  incompatible paths. Either use only one system or edit scummvm.ini and set
+  there only relative paths (mintlib bug/limitation).
+
+- the default theme ("ScummVM Modern Theme Remastered") is not loaded
+  automatically on plain 8+3 filesystem. If you want to use it nevertheless,
+  choose it again in the Misc/Theme option (but you can't have the theme
+  depacked as a directory; many xml/image files use long filenames).
+
+- the talkie version of MI1 needs to be merged from two sources: first generate
+  the DOS version and then additionally also the flac version. Then convert all
+  *.flac files into *.wav and replace monkey.sof (flac) with monster.sou (DOS).
+  And of course, don't forget to set the extra path in Game options to the
+  folder where *.wav files are located! For MI2 just use the DOS version,
+  there are no CD tracks available. :(
+
+
+Future plans
+------------
+
+- aspect ratio correction for VGA/SuperVidel
+
+- unified file paths in scummvm.ini
+
+- 8bpp overlay (and get rid of all that 16bpp handling code)
+
+- profiling :) (see also https://github.com/scummvm/scummvm/pull/2382)
+
+- DSP-based sample mixer
+
+- avoid loading music/speech files (and thus slowing down everything) if muted
+
+- assembly copy routines for screen/chunky surfaces (even with SuperVidel
+  present it is not possible to use the SuperBlitter for every surface)
+
+- cached audio/video streams (i.e. don't load only "output_samples" number of
+  samples but cache, say, 1 second so disk i/o wont be so stressed)
+
+- using LDG or Thorsten Otto's sharedlibs: https://tho-otto.de/sharedlibs.php
+  for game engine plugins to relieve the huge binary size
+
+- reuse modified rects in double/triple buffer in the next frame - that way we
+  wouldn't need to refresh the whole screen in every case
+
+- add support for the TT030; this would be easily possible when I rewrite the
+  renderer with a more flexible resolution switching
+
+Closing words
+—------------
+
+I plan to open a pull request with all of my code so who knows, maybe ScummVM
+2.7.0 for Atari will be already present on the official website. :-)
```

Comment:
Unfortunately, too late... We could think about 2.7.1, though. It depends on how the graphics/ changes are resolved.

Reply:
I think 2.7.1 should stick to the rule "bugfixes only" as usually projects do. I can release 2.8-git snapshots for our community if needed. Since the unofficial 2.6.1 has been released for the Atari backend and this backend focuses on 8bpp games only, I need to provide some significant changes to make it worth releasing anyway.